LF/specialattack.js

```diff
@@ -101,6 +101,11 @@ function (livingobject, Global, Futil) {
     // Special Attack Projectiles
     1002: function (event, ITR, att, attps, rect) {
       const $ = this
+      if ($.parent.name === 'Rudolf' && $.id === 202) { // bug fix for shuriken not laying on ground
```

Comment:
What is this bug actually referring to?

Reply:
As you may see the ninja star ends at the middle in the air.
![false_normal_star](https://user-images.githubusercontent.com/67236456/120986919-a9636600-c7af-11eb-9427-2bfc653aaada.gif)

The correct demonstration should be as follow, in normal circumstances, the ninja star will disappear after it has dropped on the ground, which the logic is actually applied in the program.
![true_normal_star](https://user-images.githubusercontent.com/67236456/120987160-e92a4d80-c7af-11eb-9414-24e67a5618ed.gif)

What I have found is the frame (43 of Rudolf) lost, so I just skipped this frame for now. There maybe some other method but I have just applied the existing one.

Reply:
I also edited the character name check to character id check in the next commit